packages/aws-cdk/package.json

```diff
@@ -34,8 +34,8 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/archiver": "^2.1.3",
-    "@types/fs-extra": "^5.0.5",
+    "@types/archiver": "^3.0.0",
+    "@types/fs-extra": "^7.0.0",
```

Comment:
Why not 8.0.1 like in region-info?

Reply:
Uh. No idea. I just followed the 'update-everything' script.